vmm/src/vm.rs

```diff
@@ -1195,8 +1221,10 @@ impl Vm {
         arch::configure_system(
             &mem,
             arch::layout::CMDLINE_START,
+            arch::layout::CMDLINE_MAX_SIZE,
```

Comment:
This was providing the actual cmdline length previously, but that's harder to get now, since the code changed a bit. This value only goes into the zero page for the guest kernel to interpret, and I verified the kernels see a correct cmdline. There might still be concerns about that, though.